player.h

```diff
@@ -0,0 +1,22 @@
+#ifndef PLAYER_H
+#define PLAYER_H
+#pragma once
+
+#include "ui_mainwindow.h"
+#include <QMainWindow>
+
+enum Choices { ROCK, PAPER, SCISSORS };
```

Comment:
If player does not choose anything before click go button, then it always being rock (not a good design)

Reply:
Thanks, so unordered_map is gonna fix that, right

Reply:
ok, gud lời khuyên